openmc/kinetics/__init__.py

```diff
@@ -0,0 +1,4 @@
+from openmc.kinetics.clock import *
```

Comment:
Avoid wildcard imports, according to PEP8 (these style changes are always annoying, sorry!)

> Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools. There is one defensible use case for a wildcard import, which is to republish an internal interface as part of a public API (for example, overwriting a pure Python implementation of an interface with the definitions from an optional accelerator module and exactly which definitions will be overwritten isn't known in advance).

Reply:
Sure, it wouldn't be hard for me to specify each class or function since there aren't too many. But wildcard imports are done all over OpenMC, so I'm not sure we are following that standard. See openmc/openmc/__init__.py for example.

Reply:
Fair enough!